GoSSB/Sources/raw.go

```diff
@@ -10,6 +10,7 @@ import (
 
 // ssbGetRawMessage returns a raw message JSON for the specified feed and sequence number. Feed is in the @-notation,
 // sequence starts at 1.
+//export ssbGetRawMessage
```

Comment:
You are right, I forgot about this and it worked for me as I only checked if the function works when called from Go.